app/domain/feature_gate.rb

```diff
@@ -36,27 +36,19 @@ def initialize(settings = Settings)
 
   class << self
     # Raise unless the feature is enabled
-    def assert!(feature)
-      new.assert!(feature)
-    end
+    delegate :assert!, to: :new
 
     # Raise if the feature is enabled
-    def refute!(feature)
-      new.refute!(feature)
-    end
+    delegate :refute!, to: :new
 
     # Execute the block if the feature is enabled
     def if(feature, &block)
       new.if(feature, &block)
     end
 
-    def enabled?(feature)
-      new.enabled?(feature)
-    end
+    delegate :enabled?, to: :new
 
-    def disabled?(feature)
-      new.disabled?(feature)
-    end
+    delegate :disabled?, to: :new
```

Comment:
Kleines Aufräumen am Wegesrand... :grin: 

Idealerweise hätte ich das gerne in einem Extra-Commit, da es nichts direkt mit dem Elternzugang zu tun hat. Vllt kann man das auch in einen eigenen Branch extrahieren?

Reply:
Das macht mein auto format automatisch... aber du hast natürlich recht, ich mach es rückgängig hier.